crates/bevy_color/src/hsla.rs

```diff
@@ -166,6 +172,90 @@ impl Luminance for Hsla {
     }
 }
 
+/// All color channels are added directly
+/// but alpha is blended
+///
+/// Values are not clamped
```

Comment:
Clamping is an interesting issue here. For `Srgba`, `Hsva`, etc., clamping is well defined and quite straight forward...until we consider HDR. Clamping is really dependent on what the full gamut supported by a particular monitor is. [This diagram](https://en.wikipedia.org/wiki/File:CIE1931xy_gamut_comparison.svg) on Wikipedia illustrates the problem, but misses Display P3 and a few others.

I _think_ the solution would be to offer clamping by gamut, and ideally extract the gamut information from [`wgpu`](https://docs.rs/wgpu/latest/wgpu/enum.PredefinedColorSpace.html).

Reply:
That sounds reasonable but would probably exceed the scope of this PR. I will open another Issue adressing this.